tests/security/apparmor/aa_genprof.pm

```diff
@@ -0,0 +1,113 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test the profile generation utility of Apparmor
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36886, tc#1621140
+
+use strict;
+use base "consoletest";
+use testapi;
+use utils;
+use version_utils qw(sle_version_at_least is_leap is_tumbleweed);
+
+sub run {
+
+    my $aa_tmp_prof       = "/tmp/apparmor.d";
+    my $aa_genprof_allow  = qr/\(A\)llow.*\(D\)eny/m;
+    my $aa_genprof_scan   = qr/\(S\)can system/m;
+    my $aa_genprof_finish = $aa_genprof_scan;
+    my $aa_genprof_save   = qr/\(S\)ave Changes/m;
+    my $aa_genprof_filter = [$aa_genprof_allow, $aa_genprof_finish, $aa_genprof_save];
+
+    select_console 'root-console';
+
+    systemctl('restart apparmor');
+    systemctl('start auditd');
+
+    save_screenshot;
+
+    # Must disable stdout buffering to make pipe works
+    if (   sle_version_at_least('15')
```

Comment:
Please use the new `is_sle` and not the deprecated `sle_version_at_least` anymore